spec/lib/post_action_creator_spec.rb

```diff
@@ -271,18 +271,40 @@
   end
 
   describe "take_action" do
-    before { PostActionCreator.create(Fabricate(:user), post, :inappropriate) }
+    it "will hide the post" do
+      PostActionCreator
+        .new(Fabricate(:moderator), post, PostActionType.types[:spam], take_action: true)
+        .perform
+        .reviewable
+      expect(post.reload.hidden).to eq(true)
```

Comment:
Don't know if we have a convention for this, but the other tests here use the predicate matchers:

```suggestion
      expect(post.reload).to be_hidden
```